oteapi_dlite/strategies/generate.py

```diff
@@ -154,6 +163,9 @@ class DLiteStorageConfig(AttrDict):
             ),
         ),
     ] = None
+    kb_document_base_iri: Annotated[
+        str, Field(description="Base IRI or prefix for created individuals.")
+    ] = ":"
```

Comment:
Not included in tests.

Reply:
This configuration is always a string (not optional). Its default value is tested.